Classes/BITHockeyManager.h

```diff
@@ -375,6 +375,15 @@
 /// @name Environment
 ///-----------------------------------------------------------------------------
 
+
+/**
+ Enum that determines what kind of environment the application is installed and running in.
+ 
+ See `BITEnvironment`
+ */
+@property (nonatomic, readonly) BITEnvironment appEnvironment;
```

Comment:
Maybe the following is an option? https://github.com/bitstadium/HockeySDK-iOS/blob/develop/Classes/BITCrashManager.h#L45-L61 and https://github.com/bitstadium/HockeySDK-iOS/blob/develop/Classes/BITCrashManager.h#L165-L186